nfc/spi_drv.c

```diff
@@ -171,8 +170,8 @@ int nfc_spi_read(struct nfc_dev *nfc_dev, char *buf, size_t count, int timeout)
 						msecs_to_jiffies(timeout));
 
 					if (ret <= 0) {
-						pr_err("%s: timeout error\n",
-						       __func__);
+						// pr_err("%s: timeout error\n",
+						//        __func__);
```

Comment:
Why was the timeout error message commented out? 
There should be feedback to the user when a read timeout occurs.